src/app/services/toaster.service.ts

```diff
@@ -0,0 +1,33 @@
+import { computed, Injectable, Signal, signal, WritableSignal } from '@angular/core';
+import { initialToasterConfig, TOASTER_MESSAGES } from '../shared/components/epm-toaster/toaster.config';
+import { ToasterMessage } from '../app.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ToasterService {
+  private isHidden$: WritableSignal<boolean> = signal(true);
+  isHidden: Signal<boolean> = computed(this.isHidden$);
+  private toasterSettings$: WritableSignal<ToasterMessage> = signal(initialToasterConfig);
+  toasterSettings: Signal<ToasterMessage> = computed(this.toasterSettings$);
```

Comment:
format order of this propperty by our code style

Reply:
fixed

Reply:
what do you fix? in your code private and public propertyes are mixed

Reply:
I can do it only this way. But still private will remain higher.
```suggestion
  private isHidden$: WritableSignal<boolean> = signal(true);
  private toasterSettings$: WritableSignal<ToasterMessage> = signal(initialToasterConfig);
  
  isHidden: Signal<boolean> = computed(this.isHidden$);
  toasterSettings: Signal<ToasterMessage> = computed(this.toasterSettings$);
```